index.js

```diff
@@ -1,7 +1,46 @@
 // Iteration 1: Names and Input
 
+console.log("I'm ready!");
+
+let hacker1 = "Margarida";
+console.log (`The driver's name is ${hacker1}`);
+
+let hacker2 = "Guilherme";
+console.log (`The navigator's name is ${hacker2}`);
+
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log (`The driver has the longest name, it has ${hacker1.length} characters. `);
+} else if (hacker1.length < hacker2.length) {
+  console.log (`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+} else {
+  console.log (`Wow, you both have equally long names, ${hacker2.length} characters!`)
+}
+
 
 // Iteration 3: Loops
+
+
+let hacker1Name = "";
+for (let i = 0; i <= hacker1.length - 1; i++) {
+ hacker1Name = hacker1Name + " " + hacker1[i];
+}
+ console.log (hacker1Name.toUpperCase());
```

Comment:
You should use the `.toUpperCase()` method inside the loop, otherwise you are not saving the string with the upper cased letters, you are just console logging it:

```
let hacker1Name = "";

  for (let i = 0; i < hacker1.length - 1; i++) {
   hacker1Name = hacker1Name + " " + hacker1[i].toUpperCase()
  }

   console.log (hacker1Name);
```

Also, since index `[0]` starts counting from 0 and `.length` starts counting from 1, inside the for loop condition you should check if `i < hacker1.length` instead of `<=` otherwise the loop will run one more lap than what it was suppose to and you will get `undefined` on the last index